packages/medical-domain/src/domains/models/medical/datum/enums/deviceevent-subtype.enum.ts

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏filename could have been `device-event-subtype.enum.ts`